docs/auth-modes.md

```diff
@@ -228,6 +228,42 @@ Note: the ```value``` attribute value for all options is customizable by connect
 })
 ```
 
+### Database impersonation using embedded credentials (DelegationUID)
+
+This applies to databases which support a user that can delegate requests from other user(s). Databases support this by enabling the client to pass a DelegationUID to the server.
+In the case of Tableau server, you can pass the identity of the logged-in user on Tableau Server as the DelegationUID. Tableau would then pass that property to the database using driver properties and authentication is taken care of by the database. This community [article](https://community.tableau.com/docs/DOC-11137) gives more information on Database Impersonation using Embedded credentials.   
+
+For this to work on Tableau Server, user should select the option "Impersonate using embedded password" while publishing the workbook from  Tableau Desktop. Below is an example for how to pass DelegationUID in a JDBC plugin. Please note that some JDBC drivers may have a different name for this property so take a look at driver documentation for the appropriate property name.
```

Comment:
Do we have an ODBC example? (Does it even make sense with ODBC?)

Reply:
yeah, it should work fine with ODBC. 
I haven't gotten the chance to test with ODBC.